Zype/Utilities/UIUtil.m

```diff
@@ -517,5 +519,125 @@ + (BOOL)isLastPageInPages:(NSNumber *)pages WithData:(NSDictionary *)parsedObjec
     return isLastPage;
 }
 
+NSString* machineName() {
+    struct utsname systemInfo;
+    uname(&systemInfo);
+    
+    return [NSString stringWithCString:systemInfo.machine
+                              encoding:NSUTF8StringEncoding];
+}
+
+NSString* systemUserAgent() {
+    UIWebView* webView = [[UIWebView alloc] initWithFrame:CGRectZero];
```

Comment:
`UIWebView` is deprecated, use `WKWebView` instead.

Reply:
Used WKWebView instead of UIWebView.